### PR TITLE
Yoel maxi galarza/logica del front

### PR DESCRIPTION
**Pull request by YoelMaxiGalarza**

### CODE REVIEW

File: client/client/server.json
Comment (br1code) [inline client/client/server.json:1]: This file is no longer necessary. Right?

File: client/client/src/components/CustomerForm.jsx [hunk 1]
```diff
@@ -10,52 +10,50 @@ import {
...
-      console.log(`Mensaje de Mesa ${table_number}: ${message}`);
-    } else {
-      console.log(`La Mesa ${table_number} necesita atencion`);
+    event.preventDefault();
+    const requestBody = {
+      restaurantId: restaurant_data.restaurantId,
+      tableId: restaurant_data.tableId,
+      message: message,
     }
+    fetch(`http://localhost:8000/api/Notifications`, {
```
Comment (br1code) [inline client/client/src/components/CustomerForm.jsx:34]: Can you create a `SERVER_URL` variable as you did in the other component?

File: client/client/src/components/CustomerForm.jsx [hunk 2]
```diff
@@ -10,52 +10,50 @@ import {
...
-    if(message !== ""){
-      setSendAlert(true);
-    }
-    else setSendAlert(false);
-    
-  }, [message])
-  
-  
-
+  console.log(restaurant_data)
```
Comment (br1code) [inline client/client/src/components/CustomerForm.jsx:44]: When possible remove any `console.log`.

File: client/client/src/routes/staff.jsx [hunk 3]
```diff
@@ -1,12 +1,47 @@
...
+  const [restaurant_id, setRestaurantId] = useState('1');
+  
+  const intervalRef = React.useRef(null);
+
+  const location = useLocation();
+  const params = new URLSearchParams(location.search);
+  const {RESTAURANT_ID} = useParams();
+
+  const fetchRestaurantNotifications = (restaurant_id) =>{
+    fetch(`http://localhost:8000/api/Notifications/${restaurant_id}`, {
```
Comment (br1code) [inline client/client/src/routes/staff.jsx:17]: Same here.

**Review by br1code — CHANGES_REQUESTED** [review]
Looks great! Proposed some minor changes.

**Comment by YoelMaxiGalarza** [comment]
All done, deleted the server.json, create a apiConstants.js to store the `SERVER_URL` and not repeat in all the files the same line of code and deleted all the innecesary `console.log()`

**Review by br1code — APPROVED** [review]

**Pull request merged**